### ISSUE TITLE
Misleading ARIMA forecast description

### ISSUE THREAD

**Issue by belg4mit** [biolab/orange3#6872]
The "forecast steps ahead" label and documentation for ARIMA in Time Series is misleading. I read this as steps beyond the input data, but it instead appears to be steps from t=0.

**Comment by belg4mit** [comment]
This also seems to be ties to the way forecasting is implemented in Orange, and has some unfortunate consequences for the accompanying confidence intervals. If a user supplies data through t=n and wishes forecasts for t=n+1 through t=n+i, they must supply t=n+i as the forecast parameter, per above. However, this also means that the confidence interval begins at t=1, ad increases from t=1 to t=n+i, even though there is known data through t=n, resulting in extraordinarily wide confidence intervals for t>n when working with lengthier time series.

In other words, I am hoping to produce something like the first figure on this page: [https://online.stat.psu.edu/stat510/lesson/3/3.3](https://online.stat.psu.edu/stat510/lesson/3/3.3), but instead end up with something like the attached
![ARIMA forecast](https://github.com/user-attachments/assets/15ab98fa-6dac-4dce-b061-44a0ad0ea8f9)

**Comment by processo** [comment]
The forecast should be attached to the end of the known period, like in the [documentation](https://orangedatamining.com/widget-catalog/time-series/line_chart/). Can you show your workflow and some data?
